koku/api/report/queries.py

```diff
@@ -118,6 +128,9 @@ def __init__(self, parameters):
         self._offset = parameters.get_filter("offset", default=0)
         self.query_delta = {"value": None, "percent": None}
         self.query_exclusions = None
+        self._tag_column = check_unleash_for_tag_column_cost_3038(
+            self._mapper.tag_column, parameters.tenant.schema_name
+        )
```

Comment:
I really wish that this check did not live here. I understand we don't have the parameters in the ProviderMap, but that's where this thing should live. The `mapper` is responsible for the tag_column, not the query handler.

I think if we want the ability to control mapper values via unleash, then we need to update all the mappers to accept the schema. Then we can define flags in specific mappers when we need to. All of this work here is specific to OCP, so it doesn't make sense to me to add it to the base report query handler which is used by all providers.